web-api/src/business/useCases/trialSessions/updateTrialSessionInteractorHelper.ts

```diff
@@ -0,0 +1,287 @@
+import { AuthUser } from '@shared/business/entities/authUser/AuthUser';
+import { Case } from '@shared/business/entities/cases/Case';
+import { PDFDocument as PDFDocumentType } from 'pdf-lib';
+import {
+  RawTrialSession,
+  TrialSession,
+} from '@shared/business/entities/trialSessions/TrialSession';
+import { ServerApplicationContext } from '@web-api/applicationContext';
+import { TRIAL_SESSION_PROCEEDING_TYPES } from '@shared/business/entities/EntityConstants';
+import { TrialSessionWorkingCopy } from '@shared/business/entities/trialSessions/TrialSessionWorkingCopy';
+import { get } from 'lodash';
+
+type GetCasesInTrialSessionParams = {
+  applicationContext: ServerApplicationContext;
+  trialSession: RawTrialSession;
+  authorizedUser: AuthUser;
+};
+
+export async function getCasesInTrialSession({
```

Comment:
A note you can ignore: Part of me wonders if it would be better to get the trial session from the db by trial session id here to avoid needing a one-off interactor to get the _counts_ of cases that need notices. (E.g., we could expose this as an interactor and check the count in the frontend.) But I can see why it was done this way if we already have the trial session info, among other reasons.